packages/wouter-preact/rollup.config.js

```diff
@@ -0,0 +1,28 @@
+import alias from "@rollup/plugin-alias";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import { defineConfig } from "rollup";
+
+export default defineConfig([
+  {
+    input: [
+      "wouter",
+      "wouter/matcher",
+      "wouter/static-location",
+      "wouter/use-location",
+    ],
+    external: ["preact", "preact/hooks"],
+
+    output: {
+      dir: "dist",
```

Comment:
Can we rename the folder to `esm`? I've seen some libraries doing that, I think it adds just a little bit of clarity.